src/forms/formHelpers.js

```diff
@@ -497,8 +506,12 @@ class ManagedNumberField extends Component {
     var { value, name, className, format }  = this.props;
 
     // Format correctly when neighbors are updated, if needed
-    if (!focused) { value = format(value); }
-    else            { value = focusedVal; }
+    if (!focused) { 
+      value = format(value); 
+    }
+    else { 
```

Comment:
I think this can go on the same line as the closing bracket above - that's how I usually see it.